### PR TITLE
Drop kartotherian_config dependency

### PR DESCRIPTION
**Pull request by GuillaumeGomez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,63 +14,53 @@ Use these commmands to download, import data and create tiles jobs for Luxembour
 **If sudo required, use "-E"!!**
 
 ```bash
-docker-compose -f docker-compose.yml -f local-compose.yml down -v
+./exec.py shutdown
```
Comment (amatissart) [inline README.md]: Maybe we could make clearer that this command will delete existing volumes (imported data, generated tiles, etc.)   
`./exec.py clean` ?

File: README.md [hunk 2]
```diff
@@ -14,63 +14,53 @@ Use these commmands to download, import data and create tiles jobs for Luxembour
...
 ```
 
 Download, import and start the tiles generation:
 
 ```bash
-docker-compose -f docker-compose.yml -f local-compose.yml up --build -d
-docker-compose -f docker-compose.yml -f local-compose.yml run --rm -e INVOKE_OSM_URL=https://download.geofabrik.de/europe/luxembourg-latest.osm.pbf -e INVOKE_TILES_X=66 -e INVOKE_TILES_Y=43 -e INVOKE_TILES_Z=7 load_db
+./exec.py load-db # it runs the build command as a dependency on 'load-db'
```
Comment (amatissart) [inline README.md]: We should explain somewhere that this command will import osm data and generate tiles on Luxembourg only, by default.

File: docker-compose.yml [hunk 3]
```diff
@@ -15,7 +15,9 @@ services:
      - "cassandra_data:/var/lib/cassandra"
 
   tilerator:
-    build: ./tilerator
+    build:
+      context: .
+      dockerfile: tilerator/Dockerfile
```
Comment (amatissart) [inline docker-compose.yml]: This change requires to update `.travis.yml` to build the docker images in the expected contexts.

File: tilerator/Dockerfile [hunk 4]
```diff
@@ -14,31 +14,47 @@ RUN apt-get update && \
...
     && cd /opt/tilerator \
     && git checkout 64b69c064aae15d474bd61da2c6bc35140f118f8 \
     && npm install --production
 
-RUN git clone https://github.com/QwantResearch/kartotherian_config.git /opt/kartotherian_config \
-    && cd /opt/kartotherian_config \
-    && git checkout c9fefaf505165c5a01bd24e3bc5295cfa9e73bde
+# install openmaptiles-tools
+RUN python3.5 -m pip install --upgrade pip \
+    && python3.5 -m pip install openmaptiles-tools
```
Comment (amatissart) [inline tilerator/Dockerfile]: The version of openmaptiles-tools should be pinned
```suggestion
    && python3.5 -m pip install openmaptiles-tools==0.9.2
```

File: README.md (hunk 2 above)
Comment (GuillaumeGomez) [reply README.md]: Done in the helper.

File: telegraf/Dockerfile [hunk 5]
```diff
@@ -1,3 +1,3 @@
 FROM telegraf:1.8-alpine@sha256:7f6873f5745fe9b4f610c5dcfaaa73d7cd75adb63bd770bb70c26a7e85c94a2a
 
-COPY telegraf.conf /etc/telegraf/telegraf.conf
+COPY telegraph/telegraf.conf /etc/telegraf/telegraf.conf
```
Comment (amatissart) [inline telegraf/Dockerfile]: typo `telegraph` :arrow_right: `telegraf`

File: exec.py [hunk 6]
```diff
@@ -0,0 +1,227 @@
...
+    except Exception as err:
+        print(f'error happened when trying to get submodules: {err}')
+    return dirs
+
+
+def init_submodule_if_not():
+    submodules = get_submodules()
+    for sub in submodules:
+        if len(os.listdir(sub)) == 0:
+            if exec_command(['git', 'submodule', 'update', '--init', sub]) != 0:
```
Comment (amatissart) [inline exec.py]: this command is failing : `exec_command` requires some `options`.  
Why not always calling using `git submodule update --init` to init all submodules ?

Comment (GuillaumeGomez) [reply exec.py]: Because in case you updated the commit, you don't want it to be overwritten if not committed.

File: exec.py [hunk 7]
```diff
@@ -0,0 +1,229 @@
...
+
+def run_load_db_france(options):
+    options['osm-file'] = "https://download.geofabrik.de/europe/france-latest.osm.pbf"
+    # got tiles from http://tools.geofabrik.de/calc/?grid=1
+    options['tiles-coords'] = '[[15, 10, 5], [16, 10, 5], [15, 11, 5], [16, 11, 5]]'
+    run_load_db(options)
+
+
+def run_update_tiles(options):
+    ret = run_load_db(options)
```
Comment (amatissart) [inline exec.py]: `run_update_tiles` should not run load_db again

**Pull request merged**